src/TraceEvent/TraceEvent.cs

```diff
@@ -171,6 +171,20 @@ public ClrTraceEventParser Clr
             }
         }
 
+        /// <summary>
+        /// For convenience, we provide a property returns a ClrPrivateTraceEventParser that knows 
+        /// how to parse all the privte Common Language Runtime (CLR .NET) events into callbacks.
+        /// </summary>
+        public ClrPrivateTraceEventParser ClrPrivate
```

Comment:
I have resisted making a shortcut for the ClrPrivate provider.   The idea is that (as its name suggests) is not public and in fact we want it to go away (move everything into the CLR provider).   

Also we only make these helpers for the TRULY important providers (kernel, clr, dynamic)., and only to help discoverability.    Arguably ClrPrivateTraceEventParser does not fall into that bucket.    

Every other provider you just need to do a 'New' on that one.  This is what we should do here.   

Remove this helper and simply new it up as needed.